ui/components/multichain/import-nfts-modal/import-nfts-modal.js

```diff
@@ -215,10 +215,7 @@ export const ImportNftsModal = ({ onClose }) => {
         <ModalContent>
           <ModalHeader
             onBack={() => setActionMode(ACTION_MODES.IMPORT_NFT)}
-            onClose={() => {
-              onClose();
-              history.push();
```

Comment:
Removed because the original code is wrong.  Calling `history.push()` without any arguments will confuse react-router-dom to have nowhere to navigate, and throw an error `(TypeError: Cannot read property 'pathname' of undefined)`